packages/driver/src/multi-domain/communicator.ts

```diff
@@ -105,9 +105,21 @@ export class PrimaryDomainCommunicator extends EventEmitter {
    * @param {string} event - the name of the event to be sent.
    * @param {any} data - any meta data to be sent with the event.
    */
-  toSpecBridge (event: string, data?: any) {
+  toAllSpecBridges (event: string, data?: any) {
+    debug('=> to all spec bridges', event, data)
```

Comment:
I added these debug messages on eventing, I find that when i'm debugging multi-domain I'm always setting these to see what communication is happening.

Reply:
Funny story: to enable these debug messages you have to add a local storage variable `localStorage.debug = 'cypress:*'`, but with session enabled, we clear local storage after each test. 🤦‍♂️